config/locales/hello.es.yml

```diff
@@ -10,63 +10,59 @@ es:
+        success: "Olvidó su contraseña"
       sign_up:
         success: "Tu cuenta ha sido creada correctamente"
         error:
-          one: "1 error ocurrió tratando de crear tu cuenta"
-          other: "%{count} errores ocurrieron tratando de crear tu cuenta"
+          one: "Ha ocurrido un error tratando de crear tu cuenta"
+          other: "Han ocurrido %{count} errores intentando crear tu cuenta"
       sign_in:
         success: "Sesión iniciada correctamente"
         error:
-          one: "1 error ocurrió tratando de inciar sesión"
-          other: "%{count} errores ocurrieron tratando de iniciar sesión"
+          one: "Ha ocurrido un error intentando iniciar sesión"
+          other: "Han ocurrido %{count} errores intentando iniciar sesión"
       forgot_password:
-        # TODO: adapt locale to Spanish
-        # success: "Hemos enviado un correo con instrucciones de como resetar tu contraseña"
-        success: "To get back into your account, follow the instructions we've sent to your <strong>\"%{login}\"</strong> email address."
+        success: "Hemos enviado un correo con instrucciones de como reestablecer tu contraseña a la dirección <strong>\"%{login}\"</strong>"
         error:
-          one: "1 error ocurrió tratando de encontrar tus credenciales"
-          other: "%{count} errores ocurrieron tratando de encontrar tus credenciales"
+          one: "Ha ocurrido un error intentado hallar tus credenciales"
+          other: "Han ocurrido %{count} errores intentando hallar tus credenciales"
       reset_password:
-        success: "Has reseteado tu contraseña satisfactoriamente"
-        alert: "Este enlace ha expirado, por favor pide un nuevo enlace"
+        success: "Tu contraseña ha sido restablecida satisfactoriamente"
+        alert: "Este enlace ha expirado, solicite uno nuevo por favor"
         error:
-          one: "1 error ocurrió mientras reseteabamos tu contraseña"
-          other: "%{count} errores ocurrieron mientras reseteabamos tu contraseña"
+          one: "Ha ocurrido un error mientras reestablecíamos su contraseña"
+          other: "Han ocurrido %{count} errores mientras reestablecíamos su contraseña"
       send_confirmation_email:
-        success: "Hemos enviado un correo de confirmación a %{email}"
+        success: "Se ha enviado un correo de confirmación a la direción %{email}"
       confirm_email:
-        success: "%{email} ha sido confirmado satisfactoriamente."
-        alert: "Este enlace ha expirado, por favor pide un nuevo enlace"
+        success: "La dirección de correo electrónico %{email} ha sido confirmada satisfactoriamente"
+        alert: "Este enlace ha expirado, solicite uno nuevo por favor."
       sudo_mode_authentication:
-        success: "Ahora sabemos que realmente eres tu. No preguntaremos tu contraseña nuevamente por 60 minutos"
+        success: "Hemos confirmado que realmente eres tú, tu contraseña no será solicitada nuevamente en los próximos 60 minutos"
```

Comment:
We need to agree on using **su** or **tu** to make a consistent translation, IMHO both are equally valid, but I'd like to see consistency across all the translated sentences.

Let me explain what do I mean with an example.
If we agreed on using **su** (a more formal Spanish), the translation will be:
* _Hemos confirmado que realmente es usted, su contraseña no será solicitada nuevamente en los próximos 60 minutos._

Otherwise, if we agreed on using **tu**, the translation would be as follows:
* _Hemos confirmado que realmente eres tú, tu contraseña no será solicitada nuevamente en los próximos 60 minutos_.